src/services/llms/base.py

```diff
@@ -0,0 +1,13 @@
+from abc import ABC, abstractmethod
+from typing import List, Dict, Any
+from ...core.models import PRDetails
+
+
+class BaseLLMService(ABC):
```

Comment:
The `BaseLLMService` class is defined but lacks implementation details.  Consider adding a docstring to explain its purpose and how it interacts with other parts of the system.  Also, providing example implementations or outlining expected behaviours would significantly aid understanding and future development.  For example, what should happen if `file_path` or `diff_content` are invalid in `create_prompt`?